README.quickstart.md

```diff
@@ -1,29 +1,27 @@
 Easy-RSA 3 Quickstart README
 ============================
 
-This is a quickstart guide to using Easy-RSA version 3. Detailed help on usage
-and specific commands by running easyrsa with the 'help' command. Additional
-documentation can be found in the doc/ directory.
+This is a quickstart guide to using Easy-RSA version 3. Detailed help on usage and specific commands can be found by running `./easyrsa -h`.
+Additional documentation can be found in the doc/ directory.
 
-If you're upgrading from the Easy-RSA 2.x series there are Upgrade-Notes
+If you're upgrading from the Easy-RSA 2.x series, there are Upgrade-Notes
```

Comment:
If you're going to convert other lines to longer than 80 characters, you should be consistent throughout the document.